setuptools/dist.py

```diff
@@ -731,13 +731,13 @@ def finalize_options(self):
         to influence the order of execution. Smaller numbers
         go first and the default is 0.
         """
-        hook_key = 'setuptools.finalize_distribution_options'
+        group = 'setuptools.finalize_distribution_options'
```

Comment:
Renamed `hook_key` because then the `eps = ...`  below fits on one line and it's more accurate anyways.